src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const calculator = {
+    result: 0,
+    add(a) {
+      calculator.result = calculator.result + a;
+
+      return calculator.result;
+    },
+    subtract(a) {
```

Comment:
```suggestion
    },
    
    subtract(a) {
```
Добавляй пробелы между методами, так легче читать код)